lib/server.js

```diff
@@ -187,10 +187,10 @@ function Server(options) {
 
     this.router.on('mount', this.emit.bind(this, 'mount'));
 
-    if (!options.handleUpgrades && PROXY_EVENTS.indexOf('upgrade') === -1) {
-        PROXY_EVENTS.push('upgrade');
+    if (!options.handleUpgrades && this.proxyEvents.includes === false) {
```

Comment:
Do you mean `this.proxyEvents.includes('upgrade') === false`? That said, I don't think node 4 supports the includes method. @hekike we may have to stick with indexOf here for now?

Reply:
You are right, apologize for extra work :/